### PR TITLE
[BUGFIX] Fix NightlyTestForBinary in master branch

### PR DESCRIPTION
**Pull request by barry-jin**
## Description ##
Fix NightlyTestForBinary in master branch. 

### Essentials ###
- [x] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by barry-jin** [comment]
@mxnet-bot run ci [centos-cpu]

**Comment by barry-jin** [comment]
@mxnet-bot run ci [centos-gpu, miscellaneous, unix-cpu, unix-gpu, website]

**Comment by barry-jin** [comment]
@mxnet-bot run ci [centos-gpu, unix-cpu, unix-gpu, website]

**Comment by barry-jin** [comment]
@mxnet-bot run ci [centos-gpu]

**Comment by barry-jin** [comment]
@mxnet-bot run ci [website, unix-cpu]

**Comment by barry-jin** [comment]
@mxnet-bot run ci [website, unix-cpu]

File: python/mxnet/gluon/contrib/estimator/batch_processor.py [hunk 1]
```diff
@@ -99,7 +100,9 @@ def fit_batch(self, estimator,
             pred = [estimator.net(x) for x in data]
             loss = [estimator.loss(y_hat, y) for y_hat, y in zip(pred, label)]
 
-        for l in loss:
-            l.backward()
+            for l in loss:
+                l.backward()
```
Comment (leezu) [inline python/mxnet/gluon/contrib/estimator/batch_processor.py:104]: You changed the indent

```suggestion
        for l in loss:
            l.backward()
```

**Comment by barry-jin** [comment]
@mxnet-bot run ci [miscellaneous, website, unix-cpu, unix-gpu]

**Comment by barry-jin** [comment]
@mxnet-bot run ci [website, unix-cpu, unix-gpu]

**Comment by barry-jin** [comment]
@mxnet-bot run ci [website, unix-cpu, unix-gpu]

**Comment by barry-jin** [comment]
@mxnet-bot run ci [website, unix-cpu, unix-gpu]

**Comment by barry-jin** [comment]
@mxnet-bot run ci [website, unix-cpu, unix-gpu]

**Comment by barry-jin** [comment]
@mxnet-bot run ci [website]

**Review by szha — APPROVED** [review]

**Pull request merged**